.changes/tinymce/7.0.0.md

```diff
@@ -0,0 +1,66 @@
+## 7.0.0 - 2024-02-29
+
+### Added
+- New custom tooltip functionality, tooltip will be shown when hovering with a mouse or with keyboard focus. #TINY-9275
+- New `sandbox_iframes_exclusions` option that holds a list of URL host names to be excluded from iframe sandboxing when `sandbox_iframes` is set to `true`. #TINY-10350
+- Added 'getAllEmojis' api function to the emoticons plugin. #TINY-10572
+- Element preset support for the `valid_children` option and Schema.addValidChildren API. #TINY-9979
+- A new `trigger` property for block text pattern configurations, allowing pattern activation with either Space or Enter keys. #TINY-10324
+- Added onFocus callback for CustomEditor dialog component. #TINY-10596
+- Added icons for the import from Word, export to Word and export to PDF premium plugins. #TINY-10612
+- Added `data` to Schema as a valid element. #TINY-10611
+- More advanced schema config for custom elements. #TINY-9980
+- Added custom tooltip for autocompleter, now visible on both mouse hover and keyboard focus, except single column cases. #TINY-9638
+- Added new `license_key` option that can be set to `gpl` or a valid license key. #TINY-10681
+- Added importword, exportpdf and exportword menu items to default file menu. #TINY-10670
```

Comment:
Fact check please, does this mean i will see these menu items by default (vanilla out of the box TinyMCE)
or when the exports plugin has been configured?

Reply:
only when configured